packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -365,6 +367,7 @@ export const bookingsRouter = router({
           eventType: {
             ...booking.eventType,
             recurringEvent: parseRecurringEvent(booking.eventType?.recurringEvent),
+            metadata: EventTypeMetaDataSchema.parse(booking.eventType?.metadata || {}),
```

Comment:
When querying for bookings, we need something in the metadata field of the booking for `getPaymentAppData`